### DISCUSSION TITLE
Download Helper for Chrome - failed downloads

### DISCUSSION THREAD

**Discussion by harrgo** [aclap-dev/video-downloadhelper#1302] [Help and Q&A]
### Your question

Sometimes when I initiate a download / capture I get the message (later) "DOWNLOAD FAILED"
However, since I often have several downloads in progress I have no idea WHICH download failed. 
Is there some way I can find out what download failed or is this an area the program needs to incorporate.

### Addon & CoApp version and details

Version 9.1.0.44 (stable), target mozilla, locale en-US
Companion App installed (v2.0.19) Recheck

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

You might want to check out the history feature.  That's something that was added in one of the recent betas.  It might be in the general availability release.  I wouldn't know.  I've been running VDH betas exclusively for at least 5 years.  History is new but it is DISABLED by default.  It was implemented such that you have to explicitly enable it.  This makes perfect sense.  No one can claim they didn't know VDH was keeping history & then suddenly they discover a huge log.  Anyone who is keeping a history log knows such a thing exists because they had to explicitly turn it on themselves.  So being disabled by default is 100% logical.

Also, if you're going to abbreviate your system environment information, there's 2 pieces of important information that you haven't provided.  When I do this, I always do it like this: Windows 11 Pro 64-bit version 23H2 build 22631.4249, Firefox 130.0.1 64-bit, licensed VDH 9.1.0.41 beta, CoApp 2.0.20.
